microsoft-edge/webview2/get-started/wpf.md

```diff
@@ -46,82 +68,168 @@ This tutorial requires Microsoft Visual Studio, not Microsoft Visual Studio Code
 <!-- ====================================================================== -->
 ## Step 3 - Create a single-window WebView2 app
 
-Start with a basic desktop project that contains a single main window.
+Start by creating a basic desktop project that contains a single main window.
 
-1. Open Microsoft Visual Studio. 
+1. Decide whether to create a **.NET Core/5/6** project (newer), or a **WPF App (.NET Framework)** project (older).  For more information, see:
+   * [.NET history](https://learn.microsoft.com/dotnet/core/introduction#net-history) in _What is .NET? Introduction and overview_.
```

Comment:
```suggestion
   * [.NET history](/dotnet/core/introduction#net-history) in _What is .NET? Introduction and overview_.
```

Using a relative URL avoids us having to change the domain name in the future when/if it changes.